packages/view3d/src/core/Renderer.ts

```diff
@@ -140,6 +140,7 @@ class Renderer {
     renderer.setPixelRatio(window.devicePixelRatio);
     renderer.setSize(width, height, false);
     this._canvasSize.set(width, height);
+    composer.setSize(width * window.devicePixelRatio, height * window.devicePixelRatio);
```

Comment:
You should call `composer.setSize` in `composer` itself, not inside the renderer in this case.
Add `resize` method to `postProcessing` and call `postProcessing.resize` inside `View3D`.

Reply:
I'll do it :)